docs/04-collaboration/01-sig-and-wg.md

```diff
@@ -22,13 +22,13 @@ The cross-project activities can require the creation of a new WG to address the
 
```

Comment:
There is a dead link under `contact Kyma Steering Committee leaders to verify it.`. See the [preview](https://deploy-preview-563--kyma-project-community-preview.netlify.app/community/04-collaboration/01-sig-and-wg#organize-new-si-gs-and-w-gs).

Reply:
Also, the link under `Remove the group from the list of SIGs and WGs in this document.` does not work. See the [website preview](https://deploy-preview-563--kyma-project-community-preview.netlify.app/community/04-collaboration/01-sig-and-wg#close-a-sig-or-wg).

Reply:
Unfortunately we have problems with links to the headers in the same documents :(